packages/gatsby-plugin-image/src/components/hooks.ts

```diff
@@ -72,6 +73,15 @@ export function getWrapperProps(
   }
 }
 
+export async function applyPolyfill(
+  ref: RefObject<HTMLImageElement>
+): Promise<void> {
+  if (!(`objectFitPolyfill` in window)) {
+    await import(`objectFitPolyfill`)
+  }
```

Comment:
I think webpack is smart enough to do one request? Could you check it to make sure? (if you already did was 👍)

If we can add a webpackChunkName to an import it might be easier for our users.

```suggestion
  if (!(`objectFitPolyfill` in window)) {
    await import(/* webpackChunkName: "gatsby-plugin-image-objectfit-polyfill" */ `objectFitPolyfill`)
  }
```

Reply:
Yes, I tested and it only does one request. Good idea about the chunk name